### ISSUE TITLE
Drag and drop mp4 not working

### ISSUE THREAD

**Issue by burgthom** [tagspaces/tagspaces#1786]
The drag and drop function is not working for me with mp4 files. The thumbnail image is dragged and dropped instead of the mp4 file. when I try to move mp4 files from the gallery to anywhere else only the image of the preview is dropped to the location jpg file not the mp4 file. I have tried reinstalling the application several times but always get the same result.

**Comment by sytolk** [comment]
Yes, I can confirm that DnD in gallery perspective is not implemented yet. 
@burgthom as temporary solution try Drag and Drop in Grid or List perspectives.

**Comment by burgthom** [comment]
I have tried list and grid perspectives as well, both have the same result. I remember it working in a previous version of tagspaces but it hasn't worked since once of the update. Please fix it so it works again.

**Comment by sytolk** [comment]
What is the size of the mp4 file? Can you check if Move/Copy works if you right click on mp4 file and use Move/Copy file dialog 
![C9421F29-02DD-46A0-894A-D5EDB4578701](https://user-images.githubusercontent.com/1415463/201337553-74d5c6a3-6145-4ec1-9afe-184ba64afc5e.png)
I suppose the problem is not with DnD and you try to move big file maybe ?

**Comment by burgthom** [comment]
They are clips so I don't think they are too big. 4-6 megabytes maybe.

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by sytolk** [comment]
What is OS ? Android

**Comment by burgthom** [comment]
Windows 11

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

**Comment by burgthom** [comment]
Also dnd function works within tagspaces but not outside.


Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
From: Thomas Burger ***@***.***>
Sent: Friday, November 11, 2022 8:17:57 AM
To: tagspaces/tagspaces ***@***.***>
Subject: RE: [tagspaces/tagspaces] Drag and drop mp4 not working (Issue #1786)


Windows 11



Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

**Comment by burgthom** [comment]
Correction: files will not dnd in list mode outside tagspaces at all. In grid mode it will just drop the thumbnail image.

**Comment by burgthom** [comment]
How do I fix this issue?

**Comment by uggrock** [comment]
This feature is not implemented yet in the application. If you want to have some files from for example the search the results somewhere else you can select them and open the copy/move dialog where you can select a folder where these files will be copied.
